core/embed/sec/tropic/tropic.c

```diff
@@ -369,10 +380,22 @@ static uint16_t get_kek_masks_slot(tropic_driver_t *drv) {
              : TROPIC_KEK_MASKS_PRIVILEGED_SLOT;
 }
 
+static void lt_mac_and_destroy_time(uint32_t *time) { *time += 51; }
+
+static void lt_r_mem_data_read_time(uint32_t *time) {
+  // Assuming the data size is 320 bytes
+  *time += 100;
+}
+
+static void lt_r_mem_data_write_time(uint32_t *time) {
+  // Assuming the data size is 320 bytes
+  *time += 77;
+}
```

Comment:
Are you sure that this is correct? I would expect write to take longer than read.